bin/gcs_cli.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Manipulate emulated GCS storage.
+
+# Usage: ./bin/gcs_cli.py CMD
+
+import os
+
+import click
+
+from google.auth.credentials import AnonymousCredentials
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+
+def get_endpoint_url(bucket_name):
+    base_url = os.environ["STORAGE_EMULATOR_HOST"]
+    return f"{base_url}/storage/v1/b/{bucket_name}"
+
+
+def get_client():
+    return storage.Client(credentials=AnonymousCredentials(), project="test")
+
+
+@click.group()
+def gcs_group():
+    """Local dev environment GCS manipulation script"""
+
+
+@gcs_group.command("create")
+@click.argument("bucket_name")
+def create_bucket(bucket_name):
+    """Creates a bucket
+
+    Specify BUCKET_NAME.
+
+    """
+    # https://github.com/fsouza/fake-gcs-server/blob/0c31d1573c14912fc58ae68118f9c9ece266756a/README.md?plain=1#L47
```

Comment:
```suggestion
    # README at https://github.com/fsouza/fake-gcs-server
```

I'd link to the emulator project page rather than a specific line in the README. Otherwise this line and the version specified in the docker image are tied to a specific version of emulator and we should update them both at the same time which we won't remember to do.